test_scripts/Defects_775/commonDefects.lua

```diff
@@ -0,0 +1,136 @@
+---------------------------------------------------------------------------------------------------
+-- Common module
+---------------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.defaultProtocolVersion = 2
+config.checkAllValidations = true
+
+--[[ Required Shared libraries ]]
+local actions = require("user_modules/sequences/actions")
+local utils = require('user_modules/utils')
+local test = require("user_modules/dummy_connecttest")
+local events = require('events')
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local json = require("modules/json")
+
```

Comment:
In case common file for scripts is already added so common items from scripts can be moved in common file